CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## v1.9.0
 * The [Zed Language Overview docs](docs/language/overview.md) have been split into multiple sections (#4576)
 * Add support for [user-defined operators](docs/language/statements.md#operator-statements) (#4417, #4635, #4646, #4644, #4663, #4674, #4698, #4702, #4716)
-* Add experimental support to the [`get` operator](https://github.com/brimdata/zed/blob/main/docs/language/operators/get.md) for customized methods, headers, and body (#4572)
+* Add experimental support to the [`get` operator](docs/language/operators/get.md) for customized methods, headers, and body (#4572)
```

Comment:
This one was a straight up mistake by me. Normally I'm careful to always use relative links in the CHANGELOG, then when I post these as release notes in GitHub I morph them there to absolute links pointing to the docs site. I just missed this one.